### ISSUE TITLE
[Experiments] Plugin not functioning on Android 14

### ISSUE THREAD

**Issue by ubergeek77** [Juby210/Aliucord-plugins#57]
I used the Experiments plugin for over a year on my old phone, but after switching to my new phone on Android 14, the plugin doesn't do anything anymore.

After installing the plugin, nothing I've tried will get the Developer options menu to show up like it did before. I've tried every combination of things you can think of - force closing Aliucord, toggling the plugin on and off, reinstalling, installing an older version of the plugin, clearing app data and setting it all up again, clearing the Aliucord directory and starting from scratch; the Developer options menu simply will not show up.

Nothing in the Aliucord logs indicate any issues. It just says

```
[PluginManager] Starting plugin: Experiments
[PluginManager] Started plugin: Experiments in 4 milliseconds
```

To rule out any incompatibilities, I've only installed the Experiments plugin after setting Aliucord up from scratch, so there are no plugins interfering with it.

Is there anything else I can try?

**Comment by siltyy** [comment]
Is this still an issue for you? Running into it rn, also on Android 14 (specifically GrapheneOS on the off chance that it matters).

Aliucord's debug logs also end up in logcat so I searched there hoping for something more informative, but I don't see anything new apart from the 2nd and 3rd entries, which also occur for other (working) plugins and are likely unrelated.
```
type: logcat
osVersion: google/oriole/oriole:14/UQ1A.240205.002/2024020500:user/release-keys
packageName: com.aliucord:126021
buffers: main,system,crash,events
level: verbose
filterRegex: Experiments

--------- beginning of main
1707611486.553 27497 27497 I Discord : [PluginManager] Loading plugin: Experiments
1707611486.554 27497 27497 E com.aliucord: Attempt to load writable dex file: /storage/emulated/10/Aliucord/plugins/Experiments.zip
1707611486.554 27497 27497 W ziparchive: Unable to open '/storage/emulated/10/Aliucord/plugins/Experiments.dm': No such file or directory
1707611486.918 27497 27497 I Discord : [PluginManager] Starting plugin: Experiments
1707611486.922 27497 27497 I Discord : [PluginManager] Started plugin: Experiments in 4 milliseconds
```

**Comment by ubergeek77** [comment]
Yes, this is still an issue. I didn't think to mention I was I on GrapheneOS, but apparently this is GrapheneOS specific. Don't understand why though. This plugin isn't doing anything too complicated, and all other plugins work fine.

**Comment by siltyy** [comment]
Well that's.. interesting.
I tried toggling exploit protection compatibility mode, but that didn't fix it. No clue what else could be behind this O.o
